contracts/src/bridge/SequencerInbox.sol

```diff
@@ -35,6 +36,14 @@ contract SequencerInbox is DelegateCallAware, GasRefundEnabled, ISequencerInbox
     mapping(address => bool) public isBatchPoster;
     ISequencerInbox.MaxTimeVariation public maxTimeVariation;
 
+    mapping(bytes32 => bool) public isValidKeysetHash;
+    mapping(bytes32 => uint256) public keysetHashCreationBlock;
+
+    modifier onlyRollupOwner() {
+        if (msg.sender != IRollupUserAbs(rollup).owner()) revert NotOwner(msg.sender, rollup);
```

Comment:
We might also want to allow rollup itself to call these methods

Reply:
I talked it over with Fred earlier. Personally I like removing the intermediate usage of the rollup admin facet for triggering this stuff since it adds complexity. If at some point we want to do more fine-grained owner control, I think we'll want to do that via a separate contract that owns everything